.gitignore

```diff
@@ -1,14 +1,3 @@
-# Don’t commit the following directories created by pub.
-.buildlog
-.pub/
-build/
+.pub
 packages
-
-# Or the files created by dart2js.
-*.dart.js
-*.js_
-*.js.deps
-*.js.map
-
-# Include when developing application packages.
```

Comment:
Shouldn't  `build/` be added instead. These excludes are only relevant when dart2js is used instead of pub build, no?